include/tlapack/lapack/conjugate.hpp

```diff
@@ -0,0 +1,36 @@
+/// @file conjugate.hpp
+/// @author Thijs Steel, KU Leuven, Belgium
+//
+// Copyright (c) 2021-2023, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#ifndef TLAPACK_CONJUGATE_HH
+#define TLAPACK_CONJUGATE_HH
+
+#include "tlapack/base/constants.hpp"
+#include "tlapack/base/utils.hpp"
+
+namespace tlapack {
+
+/** Conjugates a vector
+ *
+ * @param[in] x Vector of size n.
+ *
+ * @ingroup auxiliary
+ */
+template <class vector_t>
+void conjugate(const vector_t& x)
+{
+    using idx_t = size_type<vector_t>;
+
+    for (idx_t i = 0; i < size(x); ++i) {
+        x[i] = conj(x[i]);
```

Comment:
The issue is that `std::conj( x )` always returns a complex number regardless if x is real or complex. So, add `using tlapack::conj;` to this function. It is explained in `utils.hpp`.

Reply:
The problem wasn't the std::conj, but rather that the vector was declared as const, so of course I couldn't modify it :facepalm: Those are a few hours of my life I'll never get back.